app/helpers/host_helper/textual_summary.rb

```diff
@@ -26,7 +26,7 @@ def textual_group_relationships
       _("Relationships"),
       %i(
         ems cluster availability_zone used_tenants storages resource_pools vms templates drift_history
-        physical_server network_manager custom_button_events
+        physical_server network_manager custom_button_events cloud_networks cloud_subnets
```

Comment:
@andyvesel maybe you can use something like this (if you are not going to use support mixin ):

```ruby
   def textual_group_relationships
+    additions = []
+    additions.push(:cloud_subnets) if @record.respond_to?(:cloud_subnets)
+    additions.push(:cloud_networks) if @record.respond_to?(:cloud_networks)
+
     TextualGroup.new(
       _("Relationships"),
       %i(
         ems cluster availability_zone used_tenants storages resource_pools vms templates drift_history
-        physical_server network_manager custom_button_events cloud_networks cloud_subnets
-      )
+        physical_server network_manager custom_button_events
+      ) + additions
     )
   end
```

Reply:
That's what I was trying to suggest on gitter as a UI-side hack for the issue when talking with @andyvesel . However is this the right way to fix the issue?

@lpichler : I'd suppose that the modeling on the backend side should be done is such a way that such knowledge (and testing) on the UI side should not be needed. I'd expect the relations to be defined in some base class (Host?) not in the openstack-specific one.